system_tests/tests/pearlpc.py

```diff
@@ -40,25 +40,25 @@ def setUp(self):
         self.lewis.backdoor_run_function_on_device("re_initialise")
 
     @parameterized.expand([
-        ("EM_PRESSURE" ,"EM_PRESSURE", "BUFFER1.A", "Stop", 1),
-        ("RU_PRESSURE" ,"RU_PRESSURE", "BUFFER1.B", "Stop", 1),
-        ("RE_PRESSURE" ,"RE_PRESSURE", "BUFFER1.C", 1, 1),
-        ("St_PRESSURE" ,"St_PRESSURE", "BUFFER1.D", "Stopped", 1),
-        ("BY_PRESSURE" ,"BY_PRESSURE", "BUFFER1.E", "Active", 1),
-        ("GO_PRESSURE" ,"GO_PRESSURE", "BUFFER1.F", "TRUE", 1),
-        ("AM_PRESSURE" ,"AM_PRESSURE", "BUFFER1.G", "Auto", 1),
-        ("SL_PRESSURE" ,"SL_PRESSURE", "BUFFER1.H", "ON", 1),
-        ("SF_PRESSURE" ,"SF_PRESSURE", "BUFFER1.I", 1, 1),
-        ("ER_PRESSURE" ,"ER_PRESSURE", "BUFFER1.J", 1, 1),
-        ("PRESSURE", "PRESSURE", "BUFFER1.K",  1000, 1000),
-        ("MN_PRESSURE", "MN_PRESSURE", "BUFFER2.A",  1000, 1000),
-        ("SP_PRESSURE", "SP_PRESSURE", "BUFFER2.B",  1000, 1000),
-        ("MX_PRESSURE", "MX_PRESSURE", "BUFFER2.C",  1000, 1000)
+        ("EM_PRESSURE", "EM_PRESSURE",  0, "Stop", 1),
+        ("RU_PRESSURE", "RU_PRESSURE",  1, "Stop", 1),
+        ("RE_PRESSURE", "RE_PRESSURE",  2, 1, 1),
```

Comment:
You're changes break this test so I'm updating it to get this through review to allow additional work being impeded by this review to begin. 
```suggestion
        ("RE_PRESSURE", "RE_PRESSURE",  2, "Reset complete", 1),
```